### PR TITLE
update manifests for kiali-operator read-only root filesystem support

### PR DESCRIPTION
**Pull request by eli-gc**
Corresponding helm chart PR: kiali/helm-charts#235
Resolves: kiali/kiali#6888

### CODE REVIEW

File: manifests/kiali-community/1.78.0/manifests/kiali.v1.78.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -295,6 +296,8 @@ spec:
                   value: "1"
                 - name: ANSIBLE_CONFIG
                   value: "/etc/ansible/ansible.cfg"
+                - name: ANSIBLE_LOCAL_TEMP
+                  value: "/tmp/ansible/tmp"
```
Comment (jmazzitelli) [inline manifests/kiali-community/1.78.0/manifests/kiali.v1.78.0.clusterserviceversion.yaml:300]:
```suggestion
                - name: ANSIBLE_LOCAL_TEMP
                  value: "/tmp/ansible/tmp"
                - name: ANSIBLE_REMOTE_TEMP
                  value: "/tmp/ansible/tmp"
```

File: manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml [hunk 2]
```diff
@@ -305,6 +306,8 @@ spec:
                   value: "1"
                 - name: ANSIBLE_CONFIG
                   value: "/etc/ansible/ansible.cfg"
+                - name: ANSIBLE_LOCAL_TEMP
+                  value: "/tmp/ansible/tmp"
```
Comment (jmazzitelli) [inline manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml:310]:
```suggestion
                - name: ANSIBLE_LOCAL_TEMP
                  value: "/tmp/ansible/tmp"
                - name: ANSIBLE_REMOTE_TEMP
                  value: "/tmp/ansible/tmp"
```

File: manifests/kiali-upstream/1.78.0/manifests/kiali.v1.78.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -242,6 +243,8 @@ spec:
                   value: "1"
                 - name: ANSIBLE_CONFIG
                   value: "/etc/ansible/ansible.cfg"
+                - name: ANSIBLE_LOCAL_TEMP
+                  value: "/tmp/ansible/tmp"
```
Comment (jmazzitelli) [inline manifests/kiali-upstream/1.78.0/manifests/kiali.v1.78.0.clusterserviceversion.yaml:247]:
```suggestion
                - name: ANSIBLE_LOCAL_TEMP
                  value: "/tmp/ansible/tmp"
                - name: ANSIBLE_REMOTE_TEMP
                  value: "/tmp/ansible/tmp"
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]
need to specify the remote tmp env var also

**Comment by eli-gc** [comment]
PR updated with requested changes.

**Comment by jmazzitelli** [comment]
This works. How I tested locally (hack scripts and makefiles are run from within in the kiali/kiali repo):

1. Start minikube and install Istio
```bash
hack/k8s-minikube.sh start && hack/k8s-minikube.sh istio
```
2. Build server and operator images and push them to the cluster
```bash
make -e CLUSTER_TYPE=minikube build build-ui cluster-push
```
3. Install OLM, build and push the OLM metadata for the operator, then install the operator via OLM and kiali via CR:
```bash
make -e CLUSTER_TYPE=minikube olm-install olm-operator-create kiali-create
```
4. You know everything works if you end up with a Kiali server pod in the istio-system namespace. Running this:
```bash
kubectl get pods -n istio-system -l app.kubernetes.io/name=kiali --watch=true
```
should eventually produce an output line like this:
```
kiali-649d45cb65-pvqsz   1/1     Running             0          31s
```

**Review by jmazzitelli — APPROVED** [review]
lgtm, testing passes

**Pull request merged**